lib/libc/include/any-windows-any/intrin.h

```diff
@@ -50,7 +50,7 @@
  * On GCC 4.9 we may always include those headers. On older GCCs, we may do it only if CPU
  * features used by them are enabled, so we need to check macros like __SSE__ or __MMX__ first.
  */
-#if __MINGW_GNUC_PREREQ(4, 9)
+#if __MINGW_GNUC_PREREQ(4, 9) || defined(__clang__)
```

Comment:
I wonder if this will let us try again, making gnu the default ABI for windows. This would potentially prevent zig from having to try to look for MSVC installations on windows.

Reply:
Do you have any link to the past issues?

Reply:
I poked around a bit and couldn't find anything. Sorry for the poor documentation in this way. But I remember once I got mingw-w64 hooked up, I tried to make gnu the default C ABI for windows, and ran into header file compilation errors that looked suspiciously like they would be fixed by the patch in this changeset.